src/Accounts/Accounts/Models/RMProfileClient.cs

```diff
@@ -130,21 +129,22 @@ public AzureRmProfile Login(
             bool skipValidation,
             IOpenIDConfiguration openIDConfigDoc,
             Action<string> promptAction,
+            string claimsChallenge = null,
```

Comment:
Login() has no other usage except for in the tests. So this is fine

Reply:
Yes. If don't assign with a default value, tests will fail.